packages/terra-functional-testing/src/terra-cli/wdio/test-runner.js

```diff
@@ -86,7 +86,9 @@ class TestRunner {
    * @param {string} options.externalHost - The host address the testing environment is connected to.
    * @param {number} options.externalPort - The port mapping from the host to the container.
    * @param {string} options.formFactors - A list of form factors for the test run.
+   * @param {object} options.gitApiUrl - 
    * @param {string} options.gridUrl - The remote selenium grid address.
+   * @param {string} options.issueNumber -
```

Comment:
Missing description.

Should `gitToken` and `buildUrl` be added for consistency?

Reply:
updated here https://github.com/cerner/terra-toolkit/pull/755/commits/627a4865cc4c1e83b773cfe1799d89d092b314b2#diff-f13ec26e86b6e8d765ec9020014d38600b7df8643e37a26d40000d28807e2bfbR91